components/CardsMonitoring.vue

```diff
@@ -42,22 +28,20 @@
 <script lang="ts">
 import Vue from 'vue'
 import CardRow from '@/components/cards/CardRow.vue'
-import ExternalLink from '@/components/ExternalLink.vue'
 import ConfirmedCasesDetailsCard from '@/components/cards/ConfirmedCasesDetailsCard.vue'
 import ConfirmedCasesNumberCard from '@/components/cards/ConfirmedCasesNumberCard.vue'
 import MonitoringConfirmedCasesNumberCard from '@/components/cards/MonitoringConfirmedCasesNumberCard.vue'
-import PositiveRateCard from '~/components/cards/PositiveRateCard.vue'
 import SevereCaseCard from '@/components/cards/SevereCaseCard.vue'
 import UntrackedRateCard from '@/components/cards/UntrackedRateCard.vue'
 import ConfirmedCasesIncreaseRatioByWeekCard from '@/components/cards/ConfirmedCasesIncreaseRatioByWeekCard.vue'
 import HospitalizedNumberCard from '@/components/cards/HospitalizedNumberCard.vue'
 import MonitoringStatusOverviewCard from '@/components/cards/MonitoringStatusOverviewCard.vue'
 import MonitoringConsultationDeskReportsNumberCard from '@/components/cards/MonitoringConsultationDeskReportsNumberCard.vue'
+import PositiveRateCard from '~/components/cards/PositiveRateCard.vue'
```

Comment:
表題とは関係ないのですが、気づいちゃったので修正お願いしてもいいですか？
```suggestion
import PositiveRateCard from '@/components/cards/PositiveRateCard.vue'
```